provisioning/osx/configure/src/config.spec.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import { getServices, getComposeFiles } from './config';
+
+describe('config', () => {
+  const config = {
+    stacks: [
+      {
+        name: 'services',
+        'compose-files': ['services.yml'],
+        services: [{ name: 'visualizer' }],
+      },
+      {
+        name: 'app',
+        'compose-files': ['app.yml'],
+        services: [{ name: 'rproxy', aliases: ['web'] }],
+      },
+    ],
+  };
+  it('should get services from the config', () => {
+    const expected = [
+      {
+        stack: 'services',
+        name: 'visualizer',
+        aliases: [],
+      },
+      {
+        stack: 'app',
+        name: 'rproxy',
+        aliases: ['web'],
+      },
+    ];
+    const actual = getServices(config);
+    expect(actual).to.deep.equal(expected);
```

Comment:
Just a forethought, I've grown to dislike the `expect/should` syntax as it makes it seem like natural language but it can trip you up.

I think they've fixed it in the more recent versions, but things like `expect(a).to.be.defined;` used to always pass as `.to.be.defined` was not a real assertion (even though `.to.be.undefined` was).

IIRC it now throws a error saying its not a valid property, but I still prefer the more declarative `assert` syntax where everything is a function.